### ISSUE TITLE
BBFILE_PATTERN_IGNORE_EMPTY is missing in misspell exceptions

### ISSUE THREAD

**Issue by kwavnet** [priv-kweihmann/oelint-adv#747]
https://github.com/priv-kweihmann/oelint-adv/blob/cd696962a016893f847b06b96e82198fe43c1ab7/oelint_adv/rule_base/rule_var_misspell.py#L25

is missing `BBFILE_PATTERN_IGNORE_EMPTY`.

```
BBFILE_PATTERN_IGNORE_EMPTY_meta-foo-bar = "1"
```

is still reported as a warning
